app/models/dispatcher.rb

```diff
@@ -99,10 +96,16 @@ def on_comment_created(comment)
     end
   end
 
-  def on_proposal_update(_proposal)
+  def on_proposal_update(modifier)
+    if proposal.client_slug == "ncr"
```

Comment:
ok that makes sense. do you think that checking `ncr?` here is sufficient, or should I make a larger change to clarify branching?